src/components/carousel/carousel.js

```diff
@@ -1,39 +1,48 @@
 import React, { useState } from 'react';
 import './carousel.scss';
 import Structure from '../../structure.json';
-import dfs from '../../assets/img/carousel_img_1.png';
-// import Font, { Text } from 'react-font';
-import { BsArrowLeftSquare } from 'react-icons/bs';
-import { BsArrowRightSquare } from 'react-icons/bs';
+import Image from '../../assets/img/carousel_img_1.png';
+import SlideControl from './buttons/slideControl';
 
 const Carousel = () => {
     const [slides, setSlides] = useState(Structure['carousel']);
```

Comment:
If you don't plan to setSomething you should omit setSomething part and just write const something = useState()